src/scalper.ts

```diff
@@ -701,14 +706,18 @@ export class Scalper {
       fairValue = priorFillPrice;
       console.log(this.symbol, "Fair Value Set to Prior Fill", fairValue);
     } else {
+      const chainlinkPrice = await getChainlinkPrice(new PublicKey(tokenToSplMint(this.symbol)));
       const sbPrice = await getSwitchboardPrice(new PublicKey(tokenToSplMint(this.symbol)));
       const pythPrice = await getPythPrice(new PublicKey(tokenToSplMint(this.symbol)));
       const bids = await spotMarket.loadBids(this.connection);
       const asks = await spotMarket.loadAsks(this.connection);
       const [bidPrice, _bidSize] = bids.getL2(1)[0];
       const [askPrice, _askSize] = asks.getL2(1)[0];
       const midValue = (bidPrice + askPrice) / 2.0;
-      if (sbPrice > 0) {
+      if (chainlinkPrice > 0){
+        fairValue = chainlinkPrice;
+        console.log(this.symbol, "Use Chainlink Price", chainlinkPrice);
```

Comment:
Make the logging match, so just "Chainlink Price"

Reply:
In order splicing i cleaned these logs up so this follows that convention